src/components/ChatLog.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry.js';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const chatEntryComponents = props.entries.map((chatEntry, index) => {
```

Comment:
Given that the `id` is already a unique identifier, we do not need to use `index` for the key, we can just use `id`.